configure.ac

```diff
@@ -190,6 +190,9 @@ PKG_CHECK_EXISTS([libtpms >= 0.9], [
     )
 ])
 
+AM_CONDITIONAL([WITH_TPM1], test "x$WITH_TPM1" != "x")
```

Comment:
I saw your message in the PR. I think you should drop this patch. The following should resolve the issue but we'll need to add it to all test cases:

```iff --git a/tests/common b/tests/common
index 5abe951..ee675ef 100644
--- a/tests/common
+++ b/tests/common
@@ -831,3 +831,15 @@ function validate_pidfile()
                exit 1
        fi
 }
+
+# Check whether swtpm can use a TPM 1.2
+function skip_test_no_tpm12()
+{
+       local swtpm_exe="$1"
+
+       local res=$(${swtpm_exe} socket --print-capabilities | grep '"tpm-1.2"')
+       if [ -z "${res}" ]; then
+               echo "${swtpm_exe} cannot use a TPM 1.2"
+               exit 77
+       fi
+}
diff --git a/tests/test_commandline b/tests/test_commandline
index e311bf8..d1acc7a 100755
--- a/tests/test_commandline
+++ b/tests/test_commandline
@@ -14,6 +14,8 @@ TESTDIR=${abs_top_testdir:=$(dirname "$0")}
 # need SWTPM to be set
 source ${TESTDIR}/common
 
+skip_test_no_tpm12 "${SWTPM_EXE}"
+
 TPMDIR=`mktemp -d`
 PID_FILE=$TPMDIR/${SWTPM}.pid
 LOG_FILE=$TPMDIR/${SWTPM}.log
```

Reply:
I have that patch now ...

Reply:
great, so I will pick your branch!